sonar-plugin-api/src/main/java/org/sonar/api/measures/RangeDistributionBuilder.java

```diff
@@ -41,7 +39,9 @@
  * with pre-defined ranges of complexity.</p>
  *
  * @since 1.10
+ * @deprecated since 5.2 use {@link org.sonar.api.ce.measure.RangeDistributionBuilder instead}
  */
+@Deprecated
 public class RangeDistributionBuilder implements MeasureBuilder {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)